app/Http/Resources/UKCP/AirfieldCollection.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Resources\UKCP;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class AirfieldCollection extends ResourceCollection
+{
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request): array
+    {
+        return [
+            'airfields' => $this->collection,
+        ];
+    }
+
+    /**
+     * Search for an AirfieldResource in the collection.
+     *
+     * @param string $searchTerm
+     * @return $this
+     */
+    public function search(string $searchTerm): AirfieldCollection
+    {
+        return $this->airfields->filter(fn ($item) => preg_match("/{$searchTerm}/", $this->code) || stristr($this->code, $searchTerm));
```

Comment:
Could we use Str::contains here to clean this up?

Ref: https://laravel.com/docs/8.x/helpers#method-str-contains

Reply:
done. I left the other method in as it allows to search like 'LL' => 'EGLL'